packages/flutter/lib/src/rendering/semantics.dart

```diff
@@ -312,6 +331,16 @@ class SemanticsNode extends AbstractNode {
     }
   }
 
+  Set<SemanticsTag> _tags = new Set<SemanticsTag>();
+
+  /// Tag the [SemanticsNode] with [tag].
+  void addTag(SemanticsTag tag) {
```

Comment:
It's not really needed. There is also no removeAction to accompany addAction. Actions and tags are intended to behave in the same way.

To remove a tag you never add it. That sounds strange, but you should really only add tags within a SemanticsAnotator (just like action, flags, etc). And when you get a SemanticsNode in a SemanticsAnotator it has been freshly reseted since the last time you saw it.